app-typescript/components/Carousel.tsx

```diff
@@ -1,21 +1,28 @@
 import * as React from 'react';
 import { Carousel as PrimeCarousel } from '@superdesk/primereact/carousel';
+import { Badge } from './Badge';
 
-interface IProps<T> {
+interface IImage {
+    src: string;
+    alt: string;
+}
+
+interface IProps {
     id?: string;
     className?: string;
     theme?: string;
     page?: number;
-    items: Array<T>;
+    images: Array<IImage>;
     circular?: boolean;
     autoplayInterval?: number;
     numVisible?: number;
     numScroll?: number;
     responsiveOptions?: Array<IPropsResponsive>;
-    headerTemplate?: JSX.Element;
-    footerTemplate?: JSX.Element;
-    itemTemplate(item: any): JSX.Element;
     onPageChange?(e: {page: number}): void;
+    title?: string;
+    imageCount?: number;
```

Comment:
Why not use `images.length`?

I'd be good to reorder the props, so the mandatory ones are at the top.

Reply:
I see. Can you add comments to the interface so it's clear how to use it? I assume if that prop is not passed, it would use `images.length`.